src/dataExplorer/components/Results.tsx

```diff
@@ -269,12 +275,12 @@ const NOT_SUPPORTED_GRAPH_TYPES = [
 ]
 const GraphHeader: FC = () => {
   const {view, setView} = useContext(ResultsViewContext)
-  const {result} = useContext(ResultsContext)
   const {result: subQueryResult} = useContext(ChildResultsContext)
   const {launch, clear: closeSidebar} = useContext(SidebarContext)
 
   const dataExists =
-    !!result?.parsed && result.parsed.resultColumnNames.length > 0
+    !!subQueryResult?.parsed &&
+    subQueryResult.parsed.resultColumnNames.length > 0
```

Comment:
`dataExists` should be checked by `subQueryResult`; otherwise, the "CUSTOMIZE" button will always be disabled

<img width="750" alt="Screenshot 2023-09-06 at 3 59 47 PM" src="https://github.com/influxdata/ui/assets/14298407/a5671468-08a7-46ed-852f-41137aee6999">

Reply:
👍